src/raster/rasterband.rs

```diff
@@ -50,6 +50,37 @@ fn map_resample_alg(alg: &ResampleAlg) -> u32 {
     }
 }
 
+/// Wrapper type for gdal mask flags.
+/// From the GDAL docs:
+/// - `GMF_ALL_VALID`(0x01): There are no invalid pixels, all mask values will be 255. When used this will normally be the only flag set.
+/// - `GMF_PER_DATASET`(0x02): The mask band is shared between all bands on the dataset.
+/// - `GMF_ALPHA`(0x04): The mask band is actually an alpha band and may have values other than 0 and 255.
+/// - `GMF_NODATA`(0x08): Indicates the mask is actually being generated from nodata values. (mutually exclusive of `GMF_ALPHA`)
+pub struct GdalMaskFlags(i32);
+
+impl GdalMaskFlags {
+    const GMF_ALL_VALID: i32 = 0x01;
```

Comment:
I'm a bit surprised we don't have these in `gdal-sys`.

Reply:
i think it is a define not a const variable in the cpp code. I guess bindgen does not pick that up.

Reply:
Yeah, I'm not sure what's up, I think it should generate them.

Reply:
is bindgen able to handle defines? i can't find any examples.

Reply:
Yeah, it is: https://github.com/rust-lang/rust-bindgen/issues/1185

Reply:
so what is wrong with our bindgen config? And i guess we would need to regenerate all bindings -_-